.github/workflows/docker-build.yaml

```diff
@@ -114,7 +114,7 @@ jobs:
         #  cache-to: type=gha,mode=max
           platforms: |
             linux/amd64
-        #    linux/arm64
+            linux/arm64
```

Comment:
If you uncomment it, the builds would fail. I'm on it to fix it for all of our images, but till that please keep it disabled

Reply:
Good catch!